lib/supported_locales.dart

```diff
@@ -1,33 +1,46 @@
 import 'dart:ui';
 
 const supportedLocales = const [
+  // Base language:
```

Comment:
This is the wrong place to add new languages. Check lib/settings/available_languages.dart
This file will be removed as it is unused.

Reply:
Thx, also figured it out already.